docs-source/content/security/certificates.md

```diff
@@ -24,14 +24,31 @@ certificate(s) and key:
 ```bash
 $ kubectl create secret tls weblogic-operator-cert -n weblogic-operator-ns \
   --cert=<path-to-certificate> --key=<path-to-private-key>
+```
+
+For Helm 2.x:
 
+```bash
 $ helm get values weblogic-operator
```

Comment:
Looks like this is missing a "-n" (or the 3.x example should not have a "-n").

Reply:
It is correct  Helm 2 -n is invalid but Helm 3 use namespace therefore requires it, unless it is in default namespace